frontend/src/admin/components/sidebar/userList/userList.tsx

```diff
@@ -0,0 +1,28 @@
+import React, {useState} from "react";
+import Users from "../../../users.json";
+import {User} from "../../../models/user";
+import UserListItem from "./userListItem";
+import DropdownMenu from "../dropdown/dropdown-menu";
+import './userlist.scss';
+
+export default function UserList() {
+    const [selectedUser, setSelectedUser] = useState<User>();
+    return (
+        <section className={"c-userlist"}>
+            <div className="c-userlist__header">
+                <h2>Users</h2>
+                <DropdownMenu/>
+            </div>
+            <div className="c-userlist__buttons">
+                {Users.users.map((user) => (
```

Comment:
Minor thing in this case, but generally we should avoid using implicit any types. 
I would change it to `(user: User)` here.

Reply:
This is now changed to user:User